scripts/netlify.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -ex
+
+npm run clean
+npm run build-master
```

Comment:
is this file something netlify just knows to look for, as opposed to being able to wire up an npm script?

Reply:
netlify can run any command, but i don't think i can inline the $CONTEXT check in the package.json

Reply:
you can, with some awkward bash, but also perhaps the check could go in the script itself?

Reply:
The idea was to separate the netlify part from the build process part, which can also be used sans-netlify.